src/components/plots/Header.jsx

```diff
@@ -171,8 +172,12 @@ const Header = (props) => {
     </Row>
   );
 };
+Header.defaultProps = {
+  testing: false,
+};
 
 Header.propTypes = {
+  testing: PropTypes.bool,
```

Comment:
I see that you are using this to identify if the component is called from a unit test or not, but this doesn't look right to me. Is the use case here that you want to render line 108 in this file for unit tests? What gets rendered for the unit tests if you removed this testing flag?

Reply:
It calls an api endpoint, which fails if the file is ran while testing, thats why I added the `testing` flag.

Reply:
Rather than this flag, it is better if we have that api call mocked, so that we avoid adding complexity to the production code. Search for an example in our code using `fetchMock.doMock();` Please, ping me if you we need help with it.